content/docs/tools-ui/ui-dev-messages.md

```diff
@@ -0,0 +1,537 @@
+---
+linktitle: UI Messages
+title: Design Guidelines for UI Messages in Open 3D Engine
+description: Learn about the design models and best practices for O3DE UI messages, including error messages and notifications.
+weight: 500
+toc: true
+---
+
+Are you developing user interface (UI) components that extend **Open 3D Engine (O3DE)**? Learn how to craft meaningful and useful user interface (UI) messages that help your users by reviewing the principles and best practices in this topic. UI messages indicate the status of a component, such as error, failure, warning, information, and success messages. This document covers various types of messaging including *Toast*, *Inline Notification*, *Dialog*, *Console Log*, *Event Log*, *Event Table*, *Component Card*, and *Text Input Validation*.
+
+Consistent styling in O3DE messages is crucial because it reduces the cognitive load on users to know how to resolve issues in future scenarios.
+ 
+## Prerequisites
+
+To aid in your learning, we recommend opening the **Qt Control Gallery**, a Qt widget library that demonstrates UI elements and allows you to program and preview additional elements. The QT Control Gallery is built when you build O3DE and is located in the engine's `build/<platform>/bin/<configuration>` directory.
+
+For more information, refer to [O3DE Qt Control Gallery Tool](/docs/tools-ui/uidev-control-gallery/)
+
+
+## Standard Icons
```

Comment:
I moved this heading up a level (from ### to ##). I also reworded some of the sentences throughout this doc. No other major revisions made.